third_party/pcsc-lite/naclport/server_clients_management/build/js_unittests/Makefile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+TARGET := js_pcsc_lite_server_clients_management_unittests
+
+include ../../../../../../common/make/common.mk
+
+include $(COMMON_DIR_PATH)/make/js_building_common.mk
+
+include $(COMMON_DIR_PATH)/js/include.mk
+include ../../include.mk
+
+
+JS_COMPILER_INPUT_PATHS := \
+	$(PCSC_LITE_SERVER_CLIENTS_MANAGEMENT_JS_COMPILER_INPUT_DIR_PATHS) \
```

Comment:
Where does this come from? I don't understand this (but my Makefile knowledge is limited :) )

Reply:
It comes from ../../include.mk. I've added a short comment for this piece.